config-service-impl/src/main/java/org/hypertrace/config/service/store/ConfigStore.java

```diff
@@ -42,6 +43,15 @@ UpsertedConfig writeConfig(
    */
   ContextSpecificConfig getConfig(ConfigResourceContext configResourceContext) throws IOException;
 
+  /**
+   * Get the configs with the latest version for the specified resources.
+   *
+   * @param configResourceContexts
+   * @return
+   */
+  List<ContextSpecificConfig> getConfigs(Set<ConfigResourceContext> configResourceContexts)
```

Comment:
What would be the order of the configs returned? 
- Should we consider taking in a list as an input?
OR
- Should we consider returning map as an output?